src/jit/codegenxarch.cpp

```diff
@@ -5313,6 +5313,7 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
     unsigned stackAdjustBias = 0;
 
 #if defined(_TARGET_X86_)
+#if !FEATURE_EH_FUNCLETS
     //-------------------------------------------------------------------------
```

Comment:
You could move the `#if defined(_TARGET_X86_)` below the `#endif // !FEATURE_EH_FUNCLETS` to avoid nested `#if`.